front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -446,15 +400,9 @@ describe.skip('Survey builder', () => {
       .should('not.have.attr', 'disabled');
     cy.get('#project-survey-button').find('button').click({ force: true });
     cy.contains(questionTitle).should('exist');
-    cy.get('[data-cy="e2e-next-page"]').should('be.visible').click();
     cy.get('[data-cy="e2e-submit-form"]').click();
-
-    // Check that we show a success message
-    cy.get('[data-cy="e2e-survey-success-message"]').should('exist');
-    // close modal
-    cy.get('.e2e-modal-close-button').click();
-    // check that the modal is no longer on the page
-    cy.get('#e2e-modal-container').should('have.length', 0);
+    cy.get('[data-cy="e2e-page-number-1"]').should('exist');
+    cy.get('[data-cy="e2e-after-submission"]').should('exist');
   });
```

Comment:
Do we always need both these checks? Or is it enough just to check for the "after-submission" ? :)

```
 cy.get('[data-cy="e2e-page-number-1"]').should('exist');
 cy.get('[data-cy="e2e-after-submission"]').should('exist');
```

Reply:
I was wondering the same. The `e2e-after-submission` tag is present on the "back to project" so I wasn't sure if it's sufficient to know we're on the end page (what if we would add the "back to project" button to the other pages too?). So I also added this check.

I think to remember, but really not sure anymore, that I discussed this briefly with Iva and that she suggested to keep it that way.

Let me know if you still think it's better to just check for `e2e-after-submission`, and I'll make the change.

Reply:
No I think it's fine then! :relaxed: :+1: